api/v1alpha1/agent/openapi.yaml

```diff
@@ -0,0 +1,187 @@
+openapi: 3.0.1
```

Comment:
Should we structure APIs as `apis/agent/v1alpha1` and `apis/service/v1alpha1` immediately, so they can be versioned independently? Or too disruptive?